hieradata_aws/class/integration/publishing_api_db_admin.yaml

```diff
@@ -10,17 +10,17 @@ govuk_env_sync::tasks:
     database_hostname: "publishing-api-postgres"
     temppath: "/tmp/publishing_api_production"
     url: "govuk-production-database-backups"
-    path: "postgresql-backend"
+    path: "publishing-api-postgres"
```

Comment:
Just checking the commit message - should it be `/pulls for email-alert-api DB admin/pulls for publishing-api DB admin/s`?

Reply:
Oops thanks, good catch - remnants of a copy and paste